graylog2-web-interface/src/components/graylog/Navbar.jsx

```diff
@@ -0,0 +1,260 @@
+        }
+      }
+
+      @media (max-width: @grid-float-breakpoint-max) {
+        .open .dropdown-menu {
+          > li > a {
+            color: ${teinte.tertiary.uno};
+            &:hover,
+            &:focus {
+              color: ${darken(0.25, teinte.tertiary.uno)};
+              background-color: transparent;
+            }
+          }
+          > .active > a {
+            &,
+            &:hover,
+            &:focus {
+              color: ${darken(0.15, teinte.tertiary.uno)};
+              background-color: ${darken(0.065, teinte.secondary.due)};
+            }
+          }
+          > .disabled > a {
+            &,
+            &:hover,
+            &:focus {
+              color: ${teinte.secondary.tre};
+              background-color: transparent;
+            }
+          }
+        }
+      }
+    }
+
+    .navbar-toggle {
+      border-color: ${teinte.secondary.tre};
+      &:hover,
+      &:focus {
+        background-color: ${teinte.secondary.tre};
+      }
+      .icon-bar {
+        background-color: ${darken(0.25, teinte.secondary.tre)};
+      }
+    }
+
+    .navbar-collapse,
+    .navbar-form {
+      border-color: ${darken(0.065, teinte.secondary.due)};
+    }
+
+    .navbar-link {
+      color: ${teinte.tertiary.uno};
+      &:hover {
+        color: ${darken(0.25, teinte.tertiary.uno)};
+      }
+    }
+
+    .btn-link {
+      color: ${teinte.tertiary.uno};
+      &:hover,
+      &:focus {
+        color: ${darken(0.25, teinte.tertiary.uno)};
+      }
+      &[disabled],
+      fieldset[disabled] & {
+        &:hover,
+        &:focus {
+          color: ${teinte.secondary.tre};
+        }
+      }
+    }
+  }
+
+  &.navbar-inverse {
+    background-color: ${teinte.primary.tre};
+    border-color: ${darken(0.15, teinte.primary.tre)};
+
+    .navbar-brand {
+      color: ${lighten(0.15, teinte.tertiary.uno)};
+      &:hover,
+      &:focus {
+        color: ${teinte.primary.due};
+        background-color: transparent;
+      }
+    }
+
+    .navbar-text {
+      color: ${lighten(0.15, teinte.primary.tre)};
+    }
+
+    .navbar-nav {
+      > li > a {
+        color: ${teinte.secondary.due};
+
+        &:hover,
+        &:focus {
+          color: ${teinte.tertiary.due};
+          background-color: transparent;
+        }
+      }
+      > .active > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${teinte.primary.due};
+          background-color: ${lighten(0.20, teinte.primary.tre)};
+        }
+      }
+      > .disabled > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${lighten(0.50, teinte.primary.tre)};
+          background-color: transparent;
+        }
+      }
+
+      > .open > a {
+        &,
+        &:hover,
+        &:focus {
+          color: ${teinte.primary.due};
+          background-color: ${lighten(0.20, teinte.primary.tre)};
+        }
+      }
+
+      @media (max-width: 767px) {
```

Comment:
I think this is the right time to store the breakpoints in e.g. the theme as well. What do you think? We would still need to unify them with the bootstrap variables, but it's an improvement, compared to hard coded values in the styles.

Reply:
Sounds good